clang/lib/CConv/ProgramInfo.cpp

```diff
@@ -751,17 +754,59 @@ ProgramInfo::getStaticFuncConstraintSet(std::string FuncName,
   return nullptr;
 }
 
-bool ProgramInfo::computePointerDisjointSet() {
-  ConstraintDisjointSet.Clear();
-  CVars WildPtrs;
+class ReachableVarVisitor : public boost::default_bfs_visitor {
```

Comment:
Can you add a comment that explains what this is doing? Also explain the role of the "interim constraint state" and what the invariants are about it at any given time?

Reply:
Added.